doc/modchannels.adoc

```diff
@@ -0,0 +1,82 @@
+= Modchannels
+
+allows server side mods(SSMs) comunication to client side mods(CSMs) and vice versa.
+
+== Server Side API
+
+=== Functions
+
+==== minetest.mod_channel_join(channel_name)
+
+* `channel_name`: string, modchannel name
+* returns ref for use with <<Methods>>
+* creates channel if not existing and joins the server on that channel
+* listen for messages on this channel with `minetest.register_on_modchannel_message`
```

Comment:
Why are all of these bullet points, and in the same paragraph? I suppose that it does do all of these things, but this feels like a very "not plain English" way of describing it.

Maybe something like:
```asciidoc
==== `minetest.mod_channel_join(channel_name)
* `channel_name`: `string`

Returns a new mod channel object with <<Methods>> for interacting with the channel.

This will join an existing channel named `channel_name` if it exists, or create (and then join) it as a new channel if it does not.
```

Reply:
https://github.com/minetest/minetest_docs/pull/15/commits/ef9ef4d1b5a87999062fe2a04c838f3a1b97b433